rules.adoc

```diff
@@ -775,7 +775,7 @@ ball.
 [[regulations-inference-in-lightweight]]
 ==== Infrared interference in Lightweight
 
-In Lightweight, the robot must not emit infrared light.
+Components designed to emit IR (e.g. IR time-of-flight LiDAR sensors) are not allowed and must be covered or removed if present at all. Any device designed to emit light of any wavelength must be mounted so that light is directed perpendicular to the playing field or be covered/removed.
```

Comment:
The rule about any light having to be perpendicular to the field I am somewhat worried about though, that would technically outlaw any status LED that is visible while you stand next to the field. I fear we'd be introducing the kind of ambiguity and potential for squabble the IR one seeks to eliminate.

Reply:
Also tucking this away under the "Infrared Interference" rule could be problematic.

Reply:
How common would there be a scenario where a status LED could not be seen from the top of the robot? As was pointed out on the forum, some teams may not realize any difference between poorly designed or malfunctioning robots and actual interference. Since any light emitting device could interfere with ball seeking, it would warrant that the rules be strict about having any light being emitted from a direction the ball could potentially emit from.

There is the "please see limitations for LWL league" part under the main interference section. Again - trying to not make the rules too lengthy.

Reply:
If it actually complies with the rule of only emitting light straight up, technically only status lasers would be allowed. That's what I'm trying to avoid people starting arguments about. As the rules already give referees/organizers the authority to require teams to tape off things that are shown to interfere in other robots (be it colored parts, lights or pretty much anything else) I think the whole"Any devices [...]" sentence can go without losing anything.